pg_search/src/bootstrap/create_bm25.rs

```diff
@@ -489,3 +501,33 @@ fn index_size(index_name: &str) -> Result<i64> {
 
     Ok(total_size as i64)
 }
+
+fn add_pg_depend_entry(index_oid: pg_sys::Oid, schema_oid: pg_sys::Oid) {
+    unsafe {
```

Comment:
```rust
// SAFETY:  The calls to [`pg_sys::recordDependencyOn`] are unsafe purely because of FFI.  
//    They operate on const pointers, which we stack allocate, and will raise their own ERRORs
//    should they fail.

Reply:
Thank you for the comment!